Base/Python/slicer/util.py

```diff
@@ -577,6 +627,7 @@ def forceRenderAllViews():
 def loadNodeFromFile(filename, filetype, properties={}, returnNode=False):
   """Load node into the scene from a file.
 
+  :raises RuntimeError: in case of failure
```

Comment:
Let's add `:raises RuntimeError: in case of failure` at the end after return (usual convention)

Reply:
According to the documentation that you linked in the previous code review, "raises" is just _before_ the :return:. I've updated the code now accordingly. Is this OK, or I should move "raises" just _after_ the :return:?

Reply:
Ideally, it should have the following form:

```

def public_fn_with_sphinxy_docstring(name, state=None):
    """This function does something.

    :param name: The name to use.
    :type name: str.
    :param state: Current state to be in.
    :type state: bool.
    :returns:  int -- the return code.
    :raises: AttributeError, KeyError

    """
    return 0

```

Copied from  https://pythonhosted.org/an_example_pypi_project/sphinx.html

Reply:
Done - "raises" now comes after "returns". All the other conversations in this PR are resolved, so if this looks OK then please approve. Thanks for the review!